src/AnalyzeRegistry.jl

```diff
@@ -36,6 +37,7 @@ struct Package
     license_files::Vector{LicenseTableEltype} # a table of all possible license files
     licenses_in_project::Vector{String} # any licenses in the `license` key of the Project.toml
     lines_of_code::Vector{LoCTableEltype}
+    contributors::Vector{String}
```

Comment:
Add to the docs?

Reply:
Yep, I still need to do it, thanks for the reminder!

Reply:
Done